lib/grizzly/options.ex

```diff
@@ -28,7 +29,12 @@ defmodule Grizzly.Options do
           database_file: Path.t() | nil,
           indicator_handler: (Grizzly.Indicator.event() -> :ok),
           rf_region: Supervisor.rf_region() | nil,
-          power_level: {Supervisor.tx_power(), Supervisor.measured_power()} | nil
+          power_level: {Supervisor.tx_power(), Supervisor.measured_power()} | nil,
+          zwave_support: atom(),
+          zwave_firmware_dir: [String.t()],
```

Comment:
```suggestion
          zwave_firmware_dir: Path.t(),
```

Looking at the code, I think this is a `Path.t()`. A string list looked weird to me even though a `Path.t()` is `IO.chardata()` so that's technically allowed.